tasks/RobotEye.tex

```diff
@@ -0,0 +1,40 @@
+\section{Robot Eye [Housekeeper]}
+The operator is going out, and the robot is helping them to get dressed.
+
+% \subsection{Focus}
+% This test focuses on object perception, manipulation, and planning.
+
+\subsection{Main Goal}
+The operator is getting dressed and announces the occasion to the robot, which must help them to pick an appropriate ensemble for the occasion.
+
+\noindent\textbf{Reward:} 1000pts (300pts per object).
+
+\subsection{Bonus rewards}
+\begin{enumerate}[nosep]
+	\item All three items correct (100pts)
+\end{enumerate}
+
+
+\subsection{Setup}
+\begin{itemize}[nosep]
+	\item \textbf{Location:} Living room
+
+	\item \textbf{Clothing:} There is a rack of clothing from which the operator must choose, with shoes in a bin on the floor.
+	\begin{itemize}[nosep]
+		\item\textit{Distractor objects}: There should be distractor objects in this set which are irrelevant to the scenarios named.
+		\item\textit{Target objects}: There should be target objects (more than one per scenario) which fit the scenarios described.
```

Comment:
The word scenario is reserved in the rulebook for the Party Host and Housekeeper Scenarios.